### PR TITLE
jquery: switch font to self-hosted Cairo

### PR DESCRIPTION
**Pull request by timmywil**
Fixes gh-446

### CODE REVIEW

File: themes/jquery/css/base.css [hunk 1]
```diff
@@ -676,14 +676,30 @@ table {
...
 	padding: .5em;
 }
 
+/* ==========================================================================
+   Cairo Font
+   ========================================================================== */
+
+@font-face {
+	font-family: Cairo;
+	src: url(fonts/Cairo/Cairo-Regular.ttf) format("truetype");
```
Comment (mgol) [inline themes/jquery/css/base.css:693]: Is there no WOFF/WOFF2 version?

File: themes/jquery/css/base.css [hunk 2]
```diff
@@ -1624,14 +1640,14 @@ nav#main ul {
...
-	font: normal normal 16px "klavika-web", "Helvetica Neue", Helvetica, Arial, Geneva, sans-serif;
+	font: normal 700 16px Cairo, "Helvetica Neue", Helvetica, Arial, Geneva, sans-serif;
 	margin-right: 2px;
 }
 
 nav#main li a {
 	color: #fff;
 	text-decoration: none;
-	padding: 6px 10px;
+	padding: 2px 10px;
```
Comment (mgol) [inline themes/jquery/css/base.css:1650]: What’s the reason for this change?

File: themes/jquery/css/base.css (hunk 1 above)
Comment (timmywil) [reply themes/jquery/css/base.css:693]: No, but from your other comment, it sounds like you prefer to remove woff everywhere.

Comment (timmywil) [reply themes/jquery/css/base.css:693]: Nvm, I misread. You wanted to just remove eot. Do we need woff, though?

Comment (mgol) [reply themes/jquery/css/base.css:693]: WOFF is the modern format supported everywhere; I think even WOFF2 works everywhere now. We should only ever need WOFF (and WOFF2, if available), other versions can be dropped.

Comment (timmywil) [reply themes/jquery/css/base.css:693]: See, I think we should only ever need open types like TTF and OTF. WOFF is not needed. Besides, Google doesn't have WOFF downloads. I can leave WOFF for fontawesome if you prefer, though.

Comment (timmywil) [reply themes/jquery/css/base.css:693]: Ok, I hadn't actually heard of WOFF. It seems it's just web-only TTF. We can convert it, but I'd have to dig into the Google Font license to see if that's allowed.

Comment (timmywil) [reply themes/jquery/css/base.css:693]: Cairo uses https://openfontlicense.org/, so it *should* be okay to convert. Can we remove TTF everywhere then?

Comment (timmywil) [reply themes/jquery/css/base.css:693]: I have confirmed that woff loads in IE9

**Comment by Krinkle** [comment]
@timmywil Can you mention somewhere the origin of the `woff` file? This make it easier to verify the source of the binary file in the future.

Perhaps also add a license file in the Cairo subdirectory, which would include the copyright attribution statement (assuming the source has one). Afaik the OFL is similar to MIT and CC-BY licenses, in so far that it allows free distribution so long as you attribute the author. Right now, the repo doesn't.

**Comment by timmywil** [comment]
Sure, but doesn't that mean we also need the fontawesome license?

**Comment by timmywil** [comment]
File size comparisons

| Font | ttf | woff | woff2 |
| ----- | -- | -- | -- |
| Cairo-Bold | 93kb | 48kb | 36kb |
| Cairo-Regular | 93kb | 47kb | 35kb |

So, a more significant difference from ttf to woff and a relatively minor difference for woff2, but I'll include it now that I have it.

**Review by mgol — APPROVED** [review]
The PR still adds TTF files. However, this makes sense as we need to allow reproducing how we created the WOFF ones. LGTM!

**Pull request merged**